Classes/Processor/FileProcessor.php

```diff
@@ -95,13 +95,38 @@ public function __construct(
      */
     public function renderFileCollection(string $table, string $fieldName, array $row, array $configuration = []): array
```

Comment:
This is a misnomer, it should be `renderFilesFromRelation`. But this is a preexisting problem and not part of this PR.